src/components/Header/Header.scss

```diff
@@ -0,0 +1,45 @@
+.head {
+  align-items: center;
+  background: #f5f6f7;
+  border-bottom: solid 1px #dfe0e1;
+  display: flex;
+  line-height: 30px;
```

Comment:
make the `height 50px;` instead, so we can have stuff be centered and everyhting